templates/base/package.json.hbs

```diff
@@ -4,11 +4,14 @@
     "@blueprintjs/icons": "3.24.0",
     "axios": "0.21.1",
     "express": "4.17.1",
+    "google-auth-library": "6.1.6",
```

Comment:
Do you need to always import these here? Can't they be added depending on which auth methods are actually chosen?